From 9c01ea5a8ab5667ad86270a83769793b97cbc9f0 Mon Sep 17 00:00:00 2001
From: Yi Zhao <yi.zhao@windriver.com>
Date: Fri, 3 Jul 2020 10:22:57 +0800
Subject: [PATCH] policy/modules/services/zebra: allow zebra_t to read nsfs_t

Fixes:
avc:  denied  { read } for  pid=242 comm="zebra" dev="nsfs"
ino=4026531992 scontext=system_u:system_r:zebra_t:s0-s15:c0.c1023
tcontext=system_u:object_r:nsfs_t:s0 tclass=file permissive=0

Upstream-Status: Pending

Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 policy/modules/services/zebra.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/modules/services/zebra.te b/policy/modules/services/zebra.te
index 2e79998b4..cbf818bf9 100644
--- a/policy/modules/services/zebra.te
+++ b/policy/modules/services/zebra.te
@@ -104,6 +104,7 @@ files_read_etc_runtime_files(zebra_t)
 
 fs_getattr_all_fs(zebra_t)
 fs_search_auto_mountpoints(zebra_t)
+fs_read_nsfs_files(zebra_t)
 
 term_list_ptys(zebra_t)
 
-- 
2.17.1

